### ISSUE TITLE
Translation issue

### ISSUE THREAD

**Issue by rrennick** [Prospress/action-scheduler#214]
Originally reported in https://github.com/woocommerce/woocommerce/issues/22098.

The place I stumbled upon is in /woocommerce/includes/libraries/action-scheduler/classes/ActionScheduler_WPCLI_QueueRunner.php
Around line 30:

```
public function __construct( ActionScheduler_Store $store = null, ActionScheduler_FatalErrorMonitor $monitor = null, ActionScheduler_QueueCleaner $cleaner = null ) {
	                if ( ! ( defined( 'WP_CLI' ) && WP_CLI ) ) {
	                        throw new Exception( __( 'The ' . __CLASS__ . ' class can only be run within WP CLI.', 'action-scheduler' ) );
	                }
```

**Comment by rrennick** [comment]
Closing as this is fixed in the latest.

**Comment by JPry** [comment]
For posterity, this was fixed with #213.
